sdk/template/Azure.Template/stress/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
I wouldn't mark this as a test support project. instead I would just rename the project to be Azure.Template.Stress.Tests.csproj and that will cause it to be treated as a test project. If we need special handling for these then we can add a new IsStressTestProject and default it based on the filename "Stress.Tests".

Reply:
How about I add `IsStressProject` (and later `IsPerfProject`), and condition this based on the name `*.Stress`?  The perf and stress tests are fairly different than the unit and functional tests, and I think it will be cleaner to consider them a sibling rather than nesting under "tests".  Similar to how "samples" and "tests" are siblings.

Reply:
Added `IsStressProject`, set by naming convention.